stable/mariadb/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ .Values.networkPolicy.apiVersion }}
+metadata:
+  name: "{{ template "fullname" . }}"
+spec:
+  podSelector:
+    matchLabels:
+      app: "{{ template "fullname" . }}"
+  ingress:
+    # Allow inbound connections
+    - from:
+      - podSelector:
+          matchLabels:
+            {{ template "fullname" . }}-client: "true"
```

Comment:
NetworkPolicy is specific to the namespace it is in. A namespace selector is only necessary when you want to explicitly allow traffic from a different namespace.